src/src/components/form/CustomInput.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div :class="inputContainerClasses">
+      <label
+      v-if="undefinedCheck(input.prefix)"
+      v-html="undefinedCheck(input?.prefix)"
+      class="input-prefix"
+      ></label>
+      <fa v-if="input.fieldIcon" :icon="fieldIconChoice" class="input-icon"></fa>
```

Comment:
Codacy has a fix for the issue: [':icon' should be on a new line.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
      <fa v-if="input.fieldIcon"
```